commercetools-models/src/main/java/io/sphere/sdk/shoppinglists/commands/updateactions/AddTextLineItem.java

```diff
@@ -0,0 +1,66 @@
+package io.sphere.sdk.shoppinglists.commands.updateactions;
+
+import io.sphere.sdk.commands.UpdateActionImpl;
+import io.sphere.sdk.models.LocalizedString;
+import io.sphere.sdk.shoppinglists.ShoppingList;
+import io.sphere.sdk.types.CustomDraft;
+import io.sphere.sdk.types.CustomFieldsDraft;
+
+import javax.annotation.Nullable;
+
+/**
+ * Adds a text line item.
+ *
+ * {@doc.gen intro}
+ *
+ * {@include.example io.sphere.sdk.shoppinglists.commands.ShoppingListUpdateCommandIntegrationTest#addTextLineItem()}
+ *
+ * @see ShoppingList#getTextLineItems()
+ */
+public final class AddTextLineItem extends UpdateActionImpl<ShoppingList> implements CustomDraft {
+    private final LocalizedString name;
+    private final Long quantity;
```

Comment:
Quantity is optional here too, so the getters and constructors should all have the nullable annotation, plus the constructor need to be adapted accordingly.